skrules/skope_rules.py

```diff
@@ -334,7 +334,10 @@ def fit(self, X, y, sample_weight=None):
                                                 self.estimators_features_):
 
             # Create mask for OOB samples
-            mask = ~samples
+            # mask = ~samples # samples is numpy array. when you put ~. it returns array([-(value_1 + 1), ..., -(value_n + 1)])
+            mask = list(set(np.arange(len(samples))).difference(samples))
```

Comment:
could you do something like `from sklearn.utils import indices_to_mask` and `mask = ~indices_to_mask(samples, n_samples)`?

Reply:
> could you do something like `from sklearn.utils import indices_to_mask` and `mask = ~indices_to_mask(samples, n_samples)`?

Thank you for your kind comment!
By the way, when I did commit again, It still got errors in travis CI(29.3 - python 3.5 environment...)